spec/integ/crypto/megolm-backup.spec.ts

```diff
@@ -340,6 +340,108 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("megolm-keys backup (%s)", (backe
             expect(afterCache.imported).toStrictEqual(1);
         });
 
+        it("Should import full backup in chunks", async function () {
+            fetchMock.get("path:/_matrix/client/v3/room_keys/version", testData.SIGNED_BACKUP_DATA);
+
+            aliceClient = await initTestClient();
+
+            const aliceCrypto = aliceClient.getCrypto()!;
+
+            // just mock this call
+            // @ts-ignore - mock a private method for testing purpose
+            aliceCrypto.importBackedUpRoomKeys = jest.fn();
+
+            await aliceClient.startClient();
+
+            // tell Alice to trust the dummy device that signed the backup
+            await waitForDeviceList();
+            await aliceCrypto.setDeviceVerified(testData.TEST_USER_ID, testData.TEST_DEVICE_ID);
+
+            const fullBackup: {
+                rooms: {
+                    [roomId: string]: {
+                        [sessionId: string]: any;
```

Comment:
Do we need `any` here? Isn't it a string?

```suggestion
                        [sessionId: string]: string;
```

Reply:
good catch, I messed up the type. It should be
```
                    [roomId: string]: {
                        sessions: {
                            [sessionId: string]: KeyBackupSession;
                        };
                    };

```